data/human/free worlds side plots.txt

```diff
@@ -11,6 +11,85 @@
+			`	"Their last escort was just taken out by pirates, and we need someone who is experienced enough to fend off or evade them. You must be well known here, because many of our militia officers here recommended you. This is for quite a significant project on <planet>, so you will be paid <payment>."`
+			choice
+				`	"What's the project? It seems to be a big enterprise."`
+				`	"That's a fair few freighters, should I be expecting any more pirates?"`
+					goto pirates
+				`	"Sorry, I'm not planning on going that way."`
+					decline
+			`	"The ships are carrying construction materials and specialized shipbuilding equipment for the extension of Zug's New Port, which has been a long time coming. It would greatly help the Free Worlds' efforts. I suppose that is why there have been pirates trying to pounce on these freighters, the equipment would be a rich prize for them."`
+				goto meet
+			label pirates
+			`	"Yes, I would expect more pirate contact, because the movement of so many large freighters would be a tempting target for them, and moreover, they are carrying valuable equipment for warship manufacturing. I'd recommend avoiding combat when possible. In these trying times, the loss of these ships would be... not ideal."`
+			label meet
+			`	"Well, will you help us escort these ships?"`
+			choice
+				`	"Of course, I'll get these freighters to <planet> safely."`
+				`	"No, I won't be able to, sorry."`
+					decline
+			`	"Wonderful! I expect that these ships will benefit the Free Worlds immensely."`
+			`	With that, he leaves you to meet with the captains of the freighters.`
+				accept
+	npc accompany save
+		government "Free Worlds"
+		personality escort timid
+		ship "Hauler II" "F.S. Pohutukawa"
+		ship "Hauler II" "F.S. Totara"
+		ship "Hauler II" "F.S. Kauri"
+		ship "Hauler III" "F.S. Rimu"
+		ship "Hauler III" "F.S. Kanuka"
+	npc
+		government "Pirate"
+		personality plunders staying
+		system "Zubenelgenubi"
+		fleet "Large Southern Pirates" 3
+	npc
+		government "Pirate"
+		personality plunders staying
+		system "Sabik"
+		fleet "Large Southern Pirates" 2
+	npc
+		government "Pirate"
+		personality plunders staying
+		system "Hadar"
+		fleet "Large Southern Pirates" 3
+	npc
+		government "Pirate"
+		personality plunders staying
+		system "Zeta Centauri"
+		fleet "Large Southern Pirates" 2
+	npc
+		government "Pirate"
+		personality plunders staying
+		system "Izar"
+		fleet "Large Southern Pirates" 2
+	on visit
+		dialog `You have reached <planet>, but you left part of the convoy behind! Better depart and wait for them to arrive in this star system.`
+	on complete
+		payment 600000
+		event "fw zug expansion underway"
+		event "fw zug expansion complete" 120
```

Comment:
```suggestion
		event "fw zug expansion complete" 120
		"assisted free worlds" ++
```

As you are assisting the free worlds, it makes sense that you get some credit for it, too. (this helps a few other missions & jobs be a little more frequent)